deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -624,64 +624,56 @@ cephObjectStores:
-#    # two pools. one erasure coded and one replicated.
-#    # You need to specify the replicated pool here in the `pool` parameter, it is
-#    # used for the metadata of the images.
-#    # The erasure coded pool must be set as the `dataPool` parameter below.
-#    dataPool: ec-data-pool
-#    pool: ec-metadata-pool
+#    parameters:
+#      # clusterID is the namespace where the rook cluster is running
+#      # If you change this namespace, also change the namespace below where the secret namespaces are defined
+#      clusterID: rook-ceph # namespace:cluster
+#      # (optional) mapOptions is a comma-separated list of map options.
+#      # For krbd options refer
+#      # https://docs.ceph.com/docs/master/man/8/rbd/#kernel-rbd-krbd-options
+#      # For nbd options refer
+#      # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
+#      # mapOptions: lock_on_read,queue_depth=1024
 #
-#    # (optional) mapOptions is a comma-separated list of map options.
-#    # For krbd options refer
-#    # https://docs.ceph.com/docs/master/man/8/rbd/#kernel-rbd-krbd-options
-#    # For nbd options refer
-#    # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
-#    # mapOptions: lock_on_read,queue_depth=1024
+#      # (optional) unmapOptions is a comma-separated list of unmap options.
+#      # For krbd options refer
+#      # https://docs.ceph.com/docs/master/man/8/rbd/#kernel-rbd-krbd-options
+#      # For nbd options refer
+#      # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
+#      # unmapOptions: force
 #
-#    # (optional) unmapOptions is a comma-separated list of unmap options.
-#    # For krbd options refer
-#    # https://docs.ceph.com/docs/master/man/8/rbd/#kernel-rbd-krbd-options
-#    # For nbd options refer
-#    # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
-#    # unmapOptions: force
+#      # RBD image format. Defaults to "2".
+#      imageFormat: "2"
 #
-#    # RBD image format. Defaults to "2".
-#    imageFormat: "2"
+#      # RBD image features, equivalent to OR'd bitfield value: 63
+#      # Available for imageFormat: "2". Older releases of CSI RBD
+#      # support only the `layering` feature. The Linux kernel (KRBD) supports the
+#      # full feature complement as of 5.4
+#      # imageFeatures: layering,fast-diff,object-map,deep-flatten,exclusive-lock
+#      imageFeatures: layering
 #
-#    # RBD image features, equivalent to OR'd bitfield value: 63
-#    # Available for imageFormat: "2". Older releases of CSI RBD
-#    # support only the `layering` feature. The Linux kernel (KRBD) supports the
-#    # full feature complement as of 5.4
-#    # imageFeatures: layering,fast-diff,object-map,deep-flatten,exclusive-lock
-#    imageFeatures: layering
-#  allowVolumeExpansion: true
-#  reclaimPolicy: Delete
+#    storageClass:
+#      # Change "rook-ceph" provisioner prefix to match the operator namespace if needed
+#      provisioner: rook-ceph.rbd.csi.ceph.com # driver:namespace:operator
```

Comment:
WIth #13622 that is about to merge, the driver name is changing. How about updating this now to hopefully avoid a merge conflict. 
```suggestion
#      provisioner: rook-ceph.rbd.csi.ceph.com # csi-provisioner-name
```